src/libs/ReportUtils.ts

```diff
@@ -1013,10 +1012,9 @@ Onyx.connect({
                 return acc;
             }
 
-            // eslint-disable-next-line @typescript-eslint/no-deprecated
-            InteractionManager.runAfterInteractions(() => {
+            if (report.preexistingReportID) {
```

Comment:
[nitpick] We already have this check in `handlePreexistingReport` so I'm not sure if we should do that here

Reply:
True, just because this is done in a loop I thought it may be better to do this early check here to avoid the function call? Not sure how effective in terms of performance this is

Reply:
Could you add a comment to make it clear why we do it here too?